### PR TITLE
Shops page

### PR DESCRIPTION
**Pull request by IvanSidorenkov**

### CODE REVIEW

File: src/components/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import styles from './Dropdown.module.scss';
+import dropdownImage from '@assets/images/svg/dropdownArrow.svg';
+
+interface IDropdown {
+  items: { id: number; category: string; }[];
```
Comment (AndreiLavrov) [inline src/components/dropdown/Dropdown.tsx:8]: I would create a separate interface for item.

File: src/components/dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const handleOpen = () => setIsOpen(prevState => !prevState);
+  const handleSelectItem = (id: number) => {
+    handleItem(id);
+    setIsOpen(false);
+  }
+
+  return (
+    <div className={styles.dropdownWrapper}>
+      <button onClick={handleOpen} className={styles.dropdownButton}>
+        <p className={styles.buttonText}>{items.find(item => item.id === activeItem)?.category}</p>
```
Comment (AndreiLavrov) [inline src/components/dropdown/Dropdown.tsx:25]: destructuring

File: src/components/dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <Image className={isOpen ? styles.dropdownImageActive : styles.dropdownImage} src={dropdownImage} alt="dropdown image" />
```
Comment (AndreiLavrov) [inline src/components/dropdown/Dropdown.tsx:26]: "image" word is redundant

File: src/components/dropdown/Dropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      </button>
+      <ul className={isOpen ? styles.itemsWrapper : styles.itemsWrapperClose}>
```
Comment (AndreiLavrov) [inline src/components/dropdown/Dropdown.tsx:28]: className={styles[isOpen ? 'itemsWrapper' : 'itemsWrapperClose']} as a variant?

File: src/pages/shops/index.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import MainSection from "@sections/shopsPage/mainSection/MainSection";
+
+const ShopsPage = () => {
+  return (
+    <>
```
Comment (AndreiLavrov) [inline src/pages/shops/index.tsx:5]: <> is for future?

**Review by AndreiLavrov — APPROVED** [review]

File: src/components/dropdown/Dropdown.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        {items.map(({ id, category}) =>
```
Comment (SiarheiYakauchuk) [inline src/components/dropdown/Dropdown.tsx:29]: { id, category } (space)

**Review by SiarheiYakauchuk — APPROVED** [review]

**Pull request merged**